lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -20,19 +26,29 @@ additional_paths_install:
 additional_paths_receive:
   kind: boolean
   config_get_token_append: '/^additional-paths receive$/'
-  config_set_append: '<state> additional-paths receive'
+  cli_nexus:
+    config_set_append: '<state> additional-paths receive'
+  cli_ios_xr:
+    config_set_append: ' <state> additional-paths receive'
```

Comment:
Since the leading whitespace is harmless on Nexus, consider just adding the space to the shared config_set_append entry rather than creating separate Nexus/XR entries that differ only in whitespace.